scripts/bootstrap.sh

```diff
@@ -7,4 +7,6 @@ systemctl start mysqld.service
 pip3 install -r /vagrant/requirements.txt
 
 chmod 755 /vagrant/app.py
+chmod 755 /vagrant/scripts/bootstrap_db.sh

```

Comment:
What is the benefit of a new file? Don't you think a function in bootstrap.sh would do more sense?

Reply:
@lioramilbaum  I thought that since every DB is gonna get inserted into this script, it may as well be a separate file, to keep it organized and readable. 
Is it redundant? if so, I'll combine the files.